.python-version

```diff
@@ -0,0 +1 @@
+3.5.2
```

Comment:
Is it used anyhow?

Reply:
It is used by pyenv to automatically use this version in this directory